Formula/odpi.rb

```diff
@@ -0,0 +1,22 @@
+class Odpi < Formula
+  desc "Oracle Database Programming Interface for Drivers and Applications"
+  homepage "https://oracle.github.io/odpi/"
+  url "https://github.com/oracle/odpi.git",
+    :using => :git,
+    :tag => "v2.1.0",
+    :revision => "855c898044cc7be20de74da7d42392787dcf8d49"
+  # version "2.1.0"
+
+  depends_on :xcode
```

Comment:
Seeing the makefile, it's far from clear why it would need Xcode specifically…

Reply:
That seemed the safest dependency as Xcode package contains the required C compiler.